src/conn/conn_tiered.c

```diff
@@ -216,6 +216,12 @@ __tier_do_operation(WT_SESSION_IMPL *session, WT_TIERED *tiered, uint32_t id, co
         if (ret == ENOENT)
             ret = 0;
         else if (ret == 0) {
+            /* Cache the flushed content into chunkcache. */
+            if (F_ISSET(&S2C(session)->chunkcache, WT_CHUNK_CACHE_FLUSHED_DATA_INSERTION)) {
```

Comment:
I can't see in this PR where the flag gets set?

Reply:
`        Config('flushed_data_cache_insertion', 'true', r'''
            enable caching of freshly-flushed data, before it is removed locally.''',
            type='boolean', undoc=True),`
            
  By default this flag is true.

Reply:
Cool - thanks. That config change wasn't in the PR.